Model/Config.php

```diff
@@ -146,7 +146,9 @@ public function getConfig($configPath, $scopeId = null, $scope = null)
         if (!$scope && $this->isSingleStoreMode()) {
             return $this->scopeConfig->getValue($configPath);
         }
-        return $this->scopeConfig->getValue($configPath, $scope ?: ScopeInterface::SCOPE_STORE, is_null($scopeId) ? $this->storeManager->getStore()->getId() : $scopeId);
+        $scopeId = ($scopeId !== null) ?: $this->getCurrentStoreId();
```

Comment:
This is incorrect & should be:
`$scopeId = ($scopeId === null) ? $this->storeManager->getStore()->getId() : $scopeId`

Reply:
Fixed, with small addition - I use $this->getCurrentStoreId().

Reply:
Fixed, with small addition - use $this->getCurrentStoreId().